app/controllers/api/community_videos_controller.rb

```diff
@@ -0,0 +1,40 @@
+module API
+  class CommunityVideosController < BaseController
+    def lookup
+      video = CommunityVideo::Retrieve.(params[:video_url])
+      serialized = video.attributes.slice(*%w[title platform channel_name thumbnail_url])
```

Comment:
I know Aron need to use the URL in the UI (it was fixed in the other branch)

```suggestion
      serialized = video.attributes.slice(*%w[title platform channel_name thumbnail_url url])
```